Library/src/androidTest/java/dev/testify/ScreenshotLifecycleTest.kt

```diff
@@ -60,11 +62,12 @@ class ScreenshotLifecycleTest {
         } catch (e: ScreenshotBaselineNotDefinedException) {
         }
 
-        assertEquals(5, observer.log.size)
+        assertEquals(6, observer.log.size)
         assertEquals("beforeAssertSame", observer.log[0])
-        assertEquals("beforeInitializeView", observer.log[1])
-        assertEquals("afterInitializeView", observer.log[2])
-        assertEquals("beforeScreenshot", observer.log[3])
-        assertEquals("afterScreenshot", observer.log[4])
+        assertEquals("applyConfiguration", observer.log[1])
+        assertEquals("beforeInitializeView", observer.log[2])
+        assertEquals("afterInitializeView", observer.log[3])
+        assertEquals("beforeScreenshot", observer.log[4])
+        assertEquals("afterScreenshot", observer.log[5])
```

Comment:
nit: add an assert for applyConfiguration?

Reply:
It's on line 67 @lwise